test/parallel_api/memory/specialized.algorithms/uninitialized_fill_destroy.pass.cpp

```diff
@@ -198,6 +198,7 @@ test_uninitialized_fill_destroy_by_type()
         invoke_on_all_policies<>()(test_destroy_n<T>(), p_begin, p_end, T(), n,
                                    ::std::is_trivial<T>());
 #endif
+        ::std::free(p_begin);
```

Comment:
`free` should be guarded by the same preprocessor conditions as `malloc`.

Reply:
I have switched to using a custom deleter with `std::unique_ptr` that frees the allocation.